src/components/label/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-label",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyLabel {
+	@Prop() color: string
+	@Prop() name: string
+	@Prop() description: string
+	@Prop({ reflect: true }) shape: "rectangle" | "rounded"
+
+	render() {
+		return (
+			<div
```

Comment:
This is a lonley div in a host element. instead of targetting a div inside the host target the host directly using 
```
<Host title={...} style={...}>
    {this.name}
</Host>
```

Reply:
\+